packages/workflow/src/errors/application.error.ts

```diff
@@ -26,7 +26,13 @@ export class ApplicationError extends Error {
 		this.extra = extra;
 
 		try {
-			const filePath = callsites()[2].getFileName() ?? '';
+			const filePath = callsites()[1].getFileName() ?? '';
```

Comment:
why were we using `[2]` before ?

Reply:
Honestly I cannot remember! On second thought, this might be too fragile - I'll look into trying to identify the correct strack trace line rather than trusting the index.